Client/mods/deathmatch/logic/luadefs/CLuaAudioDefs.cpp

```diff
@@ -436,6 +441,75 @@ int CLuaAudioDefs::GetSoundBufferLength(lua_State* luaVM)
     return 1;
 }
 
+int CLuaAudioDefs::SetSoundLooped(lua_State* luaVM)
+{
+    CClientSound*    pSound;
+    bool             bLoop;
+    CScriptArgReader argStream(luaVM);
+    if (argStream.NextIsUserDataOfType<CClientSound>())
+    {
+        argStream.ReadUserData(pSound);
+    }
+    else
+    {
+        m_pScriptDebugging->LogBadPointer(luaVM, "sound", 1);
+        lua_pushboolean(luaVM, false);
+        return false;
+    }
+    argStream.ReadBool(bLoop);
+
+    if (!argStream.HasErrors())
+    {
+        if (pSound)
+        {
+            if (CStaticFunctionDefinitions::SetSoundLooped(*pSound, bLoop))
+            {
+                lua_pushboolean(luaVM, true);
+                return 1;
+            }
+        }
+    }
+    else
+        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
+
+    lua_pushboolean(luaVM, false);
```

Comment:
Replace this line with `lua_pushnil(luaVM);`

Reply:
Setters will always return false on failure, this review is incorrect.